packages/fxa-auth-server/test/local/payments/paypal.js

```diff
@@ -489,7 +493,7 @@ describe('PayPalHelper', () => {
       const validInvoice = {
         ...mockInvoice,
         status: 'draft',
-        amount_due: 4.99,
+        amount_due: 499,
```

Comment:
This is cleanup from a previous PR to remind future folks that amount_due is an amountInCents.